modules/ROOT/partials/configuration/newline_behavior.adoc

```diff
@@ -0,0 +1,27 @@
+[[newline_behavior]]
+== `+newline_behavior+`
+
+NOTE: This option is only available for TinyMCE 6.1 and later.
```

Comment:
I mentioned this on Luke's PR but this should be a partial and included instead of being duplicated across all these changes: https://github.com/tinymce/tinymce-docs/pull/2471#discussion_r918548089

Reply:
I guess I just won't merge until that's in staging

Reply:
```suggestion
include::partial$misc/admon-requires-6.1v.adoc[]
```